ParseAligns/abyss-fixmate.cc

```diff
@@ -34,7 +34,14 @@ static const char USAGE_MESSAGE[] =
 "Usage: " PROGRAM " [OPTION]... [FILE]...\n"
 "Write read pairs that map to the same contig to the file SAME.\n"
 "Write read pairs that map to different contigs to stdout.\n"
-"Alignments may be in FILE(s) or standard input.\n"
+"Alignments may be in SAM-formatted FILE(s) or standard input.\n"
+"\n"
+"The distribution of fragment sizes calculated from read pairs\n"
+"that map to the same contig is first trimmed (0.0001 from each\n"
+"end) to remove outliers before calculating the mean and standard\n"
+"deviation. Read pairs that map to the same contig are not useful\n"
+"for scaffolding, other than for this purpose of calculating the\n"
+"empirical distribution of fragment sizes.\n"
```

Comment:
I'd prefer to move this information to documentation. The Wiki is publicly editable. Feel free to create a new page for `abyss-fixmate` at https://github.com/bcgsc/abyss/wiki

Reply:
You know, `abys-fixmate` sounds like 'fixing' mate information in an input stream, and outputting a stream with fixed content out. It is actually not what this utility is doing. It splits the input into pairs of reads which mapped to same reference contig from those which mapped to different contigs. Instead of proposing renaming of the executable (which would break 3rd-party apps) I at least wanted to make clear what this tool really does.

Reply:
When `--hist` is not specified, `abyss-fixmate` does the identical job as `samtools fixmate`, from which its name is derived. When `--hist` is specified, it removes reads that map to the same contig.

Reply:
That information could be added to the description of the `--hist` option.